components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentsUpdater.kt

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.experiments
+
+import android.content.Context
+import android.support.annotation.VisibleForTesting
+import androidx.work.*
```

Comment:
To prevent Android Studio from doing this:   In the Settings/Preferences dialog (⌘,), select Code Style | Java | Imports.  Make sure that the Use single class import option is enabled.